core/src/main/java/org/kiwix/kiwixmobile/core/search/SearchScreen.kt

```diff
@@ -320,6 +320,10 @@ private fun SearchListItem(
       .background(
         shape = RoundedCornerShape(EIGHT_DP),
         color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.05f)
+      )
+      .combinedClickable(
```

Comment:
You’ve made the whole row clickable, so the ripple shows on the full item. But we also have the openInNewTab icon at the end, which is clickable separately.

With this setup, it looks like the icon is part of the main row click area, which can confuse new users, especially because both the row and the icon have ripple effects.

To avoid this, I think we should show the row ripple only on the text/content area (before the icon), and let the icon keep its own ripple. This will make it clear that the icon is a separate action.